frontend/src/components/core/StreamlitDialog/ThemeCreator.tsx

```diff
@@ -66,6 +78,12 @@ const themeBuilder: ThemeBuilder = {
     options: Object.keys(CustomThemeConfig.FontFamily).map(font =>
       humanizeString(font)
     ),
+    getValue: (value: string, config: ThemeOptionBuilder): number =>
+      (config.options &&
```

Comment:
nit: can drop the `config.options &&` since it should always be set in this case

(also this formatting is not what I would have expected this to look like but there's no use in arguing with `prettier`)

Reply:
`options` is optional on `ThemeOptionBuilder` so it's needed to make TS happy :'(